### ISSUE TITLE
ESM Import: no such file or directory, open 'lib/migrations/migrate/stub/mjs.stub'

### ISSUE THREAD

**Issue by tomhoad** [knex/knex#4404]
# Environment

Knex version: 0.95.4
Database + version: Postgres 13
OS: Mac OS 10.14.6

# Bug

1. I have a `knexfile.mjs`:

```
const knex = {
  local: {
    client: "pg",
    connection: {
      host: config.dbUrl,
      database: config.dbName,
      user: config.dbUser,
      password: config.dbPass,
    },
    pool: {
      min: 2,
      max: 10,
    },
    migrations: {
      tableName: "knex_migrations",
      extension: "mjs",
      loadExtensions: [".mjs"],
    },
  },
}
```

I run:

```
./node_modules/.bin/knex --esm migrate:make create_users_table --env local --knexfile knexfile.mjs
```

2. Error message:

```
Using environment: local
Error: ENOENT: no such file or directory, open '/.../node_modules/knex/lib/migrations/migrate/stub/mjs.stub'
```

**Comment by vladshcherbin** [comment]
Same error, thrown if `migrations.extension` is set to `mjs`

**Comment by tlhunter** [comment]
I get the same error with:

- `knex@0.21.21`
- `node@16.3.0`

**Comment by kibertoad** [comment]
Released in 0.95.11
